src/software/pipeline/main_texturing.cpp

```diff
@@ -85,8 +85,12 @@ int main(int argc, char* argv[])
             "Texture edge padding size in pixel")
         ("flipNormals", po::value<bool>(&flipNormals)->default_value(flipNormals),
             "Option to flip face normals. It can be needed as it depends on the vertices order in triangles and the convention change from one software to another.")
-        ("maxNbImagesForFusion", po::value<int>(&texParams.maxNbImagesForFusion)->default_value(texParams.maxNbImagesForFusion),
-            "Max number of images to combine to create the final texture.")
+        ("useScore", po::value<bool>(&texParams.useScore)->default_value(texParams.useScore),
+             "Use triangles scores for multiband blending.")
```

Comment:
Not directly related to multi-band blending. 
Maybe `Use triangles scores (based on observations and re-projected areas in source images) for weighting contributions.` ?